evm/src/main/java/io/horizen/evm/LibEvmCallback.java

```diff
@@ -56,6 +56,7 @@ private static String invoke(int handle, String args) {
     static class CallbackProxy implements Callback {
         public String callback(int handle, Pointer msg) {
             try {
+                // we do not need to free the Pointer here, as it is freed on the libevm side when the callback returns
```

Comment:
in general, would be nice to check libevm/evm modules against the memory leaks

Reply:
True, not possible from the "inside" though, is it? I assume we would need some kind of benchmark and run it under a memory profiler.

Reply:
correct, let's keep it in mind after libevm separation